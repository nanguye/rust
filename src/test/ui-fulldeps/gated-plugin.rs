// aux-build:empty-plugin.rs

#![plugin(empty_plugin)]
//~^ ERROR The specified module could not be found. (os error 126)

fn main() {}
