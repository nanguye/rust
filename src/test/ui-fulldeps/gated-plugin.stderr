error: The specified module could not be found. (os error 126)
  --> $DIR/gated-plugin.rs:3:11
   |
LL | #![plugin(empty_plugin)]
   |           ^^^^^^^^^^^^

error: aborting due to previous error

